ibm/data_source_ibm_dl_port_test.go

```diff
@@ -0,0 +1,32 @@
+package ibm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccIBMDLPortDataSource_basic(t *testing.T) {
+	name := "dl_port"
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckIBMDLPortDataSourceConfig(name),
+				Check:  resource.ComposeTestCheckFunc(),
```

Comment:
you can add some attributes set checks 
https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/ibm/data_source_ibm_tg_gateways_test.go